app/build.gradle

```diff
@@ -77,6 +77,7 @@ dependencies {
     implementation composeBom
     androidTestImplementation composeBom
     implementation 'androidx.compose.material3:material3'
+    implementation 'androidx.compose.material:material:1.4.0'
```

Comment:
Material Design 3 doesn't support pull to refresh mechanism, we have to wait for it :(